java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -1163,4 +1163,121 @@ void testMultiAgg() {
       }
     }
   }
+
+  //
+  // FILTER TESTS
+  //
```

Comment:
Can be removed along with the follwoing new line IMO

Reply:
done.